lib/SPIRV/LLVMToSPIRVDbgTran.h

```diff
@@ -170,6 +174,10 @@ class LLVMToSPIRVDbgTran {
   std::unordered_map<const DICompileUnit *, SPIRVExtInst *> SPIRVCUMap;
   std::vector<const DbgVariableIntrinsic *> DbgDeclareIntrinsics;
   std::vector<const DbgVariableIntrinsic *> DbgValueIntrinsics;
+
+  inline static bool BuildIdentifierAndStoragePathGenerated{false};
+  inline static uint64_t BuildIdentifier;
+  inline static std::string StoragePath;
```

Comment:
May be it's worth considering caching not string and integer values, but pointer to an instruction itself like for VoidT, Int32T and DebugInfoNone, WDYT?

Reply:
Updated to cache ptrs to instructions.  Removed variables for string/integer values.